app/display.py

```diff
@@ -0,0 +1,7 @@
+def display(self, display_type: str) -> None:
+    if display_type == "console":
+        print(self.content)
+    elif display_type == "reverse":
+        print(self.content[::-1])
+    else:
+        raise ValueError(f"Unknown display type: {display_type}")
```

Comment:
The `display` function mirrors the functionality of the `print_book` method in the `Book` class. Ensure that the `display_type` values are consistent with the task requirements. If additional display types are expected, they should be implemented here. Otherwise, the function is correctly implemented for the specified functionality.